### ISSUE TITLE
Datepicker consistency

### ISSUE THREAD

**Issue by Dallas62** [Dogfalo/materialize#565]
All in the picture:
![capture d ecran 2015-01-30 a 00 47 54](https://cloud.githubusercontent.com/assets/3429329/5968666/870e240c-a81a-11e4-86c3-a2c60b11e074.png)

Steps:
- got to 31th January 2015
- move to next month (the cursor will be placed on 28, display 31)
- move to the next year (will display the content of the picture)

**Comment by pwmvx1289** [comment]
If 31th Jan 2015 is selected, it should keep on displaying Jan 31 2015, or that day in the month ie 31 Mar 2015 / 28 Feb 2015?

**Comment by Dallas62** [comment]
You should not be able to display "31 Feb",
and the display date should be the same as the selected or the cursor.

I had reported the issue because I think there is some working rules missing.

**Comment by Dogfalo** [comment]
Fixed in 45a809a96be690d402a285a3a080929c2c1f654a

**Comment by Dallas62** [comment]
Still display 31 Feb 

```
Steps:

got to 31th January 2015
move to next month (the cursor will be placed on 28, display 31)
```

**Comment by Dogfalo** [comment]
I'm not seeing it display Feb 31.

On Wed, Feb 25, 2015 at 3:31 PM, Tacyniak <notifications@github.com> wrote:

> Still display 31 Feb
>
> Steps:
>
> got to 31th January 2015
> move to next month (the cursor will be placed on 28, display 31)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Dogfalo/materialize/issues/565#issuecomment-76049638>.
>



-- 
Doggy sends his greetings from Mars.

**Comment by Dallas62** [comment]
![capture d ecran 2015-02-25 a 21 36 50](https://cloud.githubusercontent.com/assets/3429329/6379981/8ce47bd4-bd36-11e4-8efd-c878374e615c.png)

**Comment by Dogfalo** [comment]
Fixed in bea3f9a97ee14080fc88c6f7bd70ba2df5377e95

**Comment by Dallas62** [comment]
Tanks!
